code/clue.js

```diff
@@ -4,56 +4,191 @@ const mrGreen = {
+  color: 'yellow',
+  description: 'A dignified, dapper and dangerous military man',
+  age: 61,
+  image: 'assets/mustard.png',
+  occupation: 'Military officer'
+}
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'He has a lot of connections',
+  age: 48,
+  image: 'assets/white.png',
+  occupation: 'Servant'
 }
 
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: 'rope',
-  weight: 10
+  name: 'Rope',
+  weight: 10,
+}
+const knife = {
+  name: 'Knife',
+  weight: 10,
+}
+const candlestick = {
+  name: 'Candlestick',
+  weight: 10,
+}
+
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 10,
+}
+const poison = {
+  name: 'Poison',
+  weight: 10,
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 10,
+}
+const bat = {
+  name: 'Bat',
+  weight: 10,
+}
+const trophy = {
+  name: 'Trophy',
+  weight: 10,
+}
+const pistol = {
+  name: 'Pistol',
+  weight: 10,
+
+
 }
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+//2 NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  cMustard
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+]
 
-const rooms = []
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Biliard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'GuestHouse',
+  'Patio',
+]
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
-const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
-}
-
+const randomSelector = arrays => {
+  return arrays[Math.floor(Math.random() * arrays.length)]
+} 
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+const mystery = {
+  killer: {},
+  weapon: {},
+  room: {},
+};
 
 // This function will be invoked when you click on the killer card.
 const pickKiller = () => {
   // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+  mystery.killer = randomSelector(suspects);
+  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. 
+  document.getElementById('killerCard').style.background = mystery.killer.color;
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`;
+};
+document.getElementById("killerCard").onclick = pickKiller;
```

Comment:
☝🏽 since you have given the suspects properties and values and even an image (on line 4-55) you could also include them here to be shown on the cards just like you did with document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`;